sqdtoolz/Experiments/Experimental/ExpRabi.py

```diff
@@ -98,6 +101,20 @@ def _post_process(self, data):
         if self._param_rabi_decay_time:
             self._param_rabi_decay_time.Value = 1.0 / dpkt['decay_rate']
 
+        if self._phase == 0:
+            axis = 'X'
+        elif self._phase == np.pi/2:
+            axis = 'Y'
+        elif self._phase == np.pi:
+            axis = '-X'
+        elif self._phase == 3*np.pi/2:
+            axis = '-Y'
+        else:
```

Comment:
instead of having this else, defined "axis = None" at the starting of this if block.